src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -434,6 +434,10 @@ def forward(
 
         attn_probs = nn.functional.dropout(attn_weights, p=self.dropout, training=self.training)
 
+        # For int8 compatibility, sometimes the `attn_probs` are in `fp32`
+        if attn_probs.dtype != value_states.dtype:
+            attn_probs = attn_probs.to(value_states.dtype)
```

Comment:
No need for the test, PyTorch will do it for you and default to a noop :-)

Reply:
Ah yes I remember now you pointed me to that last time, will change it

Reply:
Fixed in [57c4271](https://github.com/huggingface/transformers/pull/20411/commits/57c427152f4552666c1e3bb2d5975084c8432ef9)